python/cudf/cudf/_lib/pylibcudf/types.pxd

```diff
@@ -17,6 +17,14 @@ from cudf._lib.pylibcudf.libcudf.types cimport (
     type_id,
 )
 
+from cudf._lib.pylibcudf.libcudf.types import \
+    interpolation as Interpolation  # no-cython-lint
+from cudf._lib.pylibcudf.libcudf.types import \
+    null_order as NullOrder  # no-cython-lint
+from cudf._lib.pylibcudf.libcudf.types import \
+    sorted as Sorted  # no-cython-lint
+from cudf._lib.pylibcudf.libcudf.types import order as Order  # no-cython-lint
+
```

Comment:
We don't want to put these imports into the pxd file. The pxd files should only have the cimports, i.e. it should only expose the pure Cython interface. If these are missing they should be added to the pyx file, which exports the Python interface.